fixtures/ios_app_with_helpers/Tuist/Dependencies.swift

```diff
@@ -1,5 +1,8 @@
 import ProjectDescription
 
-let dependencies = Dependencies([
-    .carthage(name: "Alamofire", requirement: .exact("5.3.0"), platforms: [.iOS]),
-])
+let dependencies = Dependencies(
+    carthage: [
+        .init(name: "Alamofire/Alamofire", requirement: .exact("5.0.4"), platforms: [.macOS]),
```

Comment:
Looking at the syntax, I think we can make the `init` private, and expose:

```swift
static func github(name: String, requirement: Requirement, platforms: [Platform]) -> CarthageDependency {}
```

Then this code would be more expressive:

```swift
let dependencies = Dependencies(
  carthage: [
    .github(name: "Alamofire/Alamofire", requirement: .exact("5.0.4"), platforms: [.macOS])
  ]
)
```